### PR TITLE
CASMNET-1065

### PR DESCRIPTION
**Pull request by lukebates123**
## Summary and Scope

1.2 docs update.  

## Issues and Related PRs
CASMINST-3755
CASMNET-1097
CASMNET-1065
CASMNET-1084
CASMNET-1095
CASMNET-685

Tested fresh install procedure on Vale.

### CODE REVIEW

File: operations/network/network_management_install_guide/manual_switch_config.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Manual switch configuration
+
+Some switch configuration is not generated by CANU and needs to be manually applied.
+
+#### SNMP configuration
```
Comment (cjohnson-hpe) [inline operations/network/network_management_install_guide/manual_switch_config.md:5]: Improved.
What is 'vault'?

File: operations/network/network_management_install_guide/manual_switch_config.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+SNMP is currently only used on sw-leaf-bmc switches, these credentials can be retrieved from vault.  More info on SNMP creds can be found on the [Change SNMP Credentials on Leaf Switches](../../operations/security_and_authentication/Change_SMNP_Credentials_on_Leaf_Switches.md) page.
```
Comment (cjohnson-hpe) [inline operations/network/network_management_install_guide/manual_switch_config.md:6]: Path is broken. Should be 
../../../security_and_authentication/Change_SMNP_Credentials_on_Leaf_Switches.md

File: operations/network/upgrade_planning.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+$ Create a CSM 1.2 configuration upgrade plan 
+
+ 
+Creating an upgrade plan is always going to be unique and dependent on the requirements of the upgrade path.  
+
+Some release versions of the network configuration require coupled upgrade of software to enable new software functionality or bug fixes that may add time required to do the full upgrade.  
+
+For example: in rel/1.2 to we will upgrade Aruba/Mellanox switches to newer code.
```
Comment (johren-hpe) [inline operations/network/upgrade_planning.md:8]: Extra "to" after "rel/1.2"

File: operations/network/network_management_install_guide/manual_switch_config.md (hunk 1 above)
Comment (lukebates123) [reply operations/network/network_management_install_guide/manual_switch_config.md:9]: Credential/password manager within CSM.

**Review by dle-hpe — APPROVED** [review]

**Review by denniswalker — APPROVED** [review]

**Pull request merged**